core/io/file_access_zip.cpp

```diff
@@ -147,7 +147,7 @@ unzFile ZipArchive::get_file_handle(String p_file) const {
 	return pkg;
 }
 
-bool ZipArchive::try_open_pack(const String &p_path, bool p_replace_files) {
+bool ZipArchive::try_open_pack(const String &p_path, bool p_replace_files, int p_offset = 0) {
```

Comment:
We also need to modify function signature here because it also implements the pure virtual ```try_open_pack()``` function.